packages-exp/auth-exp/src/model/public_types.ts

```diff
@@ -1173,10 +1190,44 @@ export interface EmulatorConfig {
 export interface AuthErrorMap {}
 
 /**
+ * The dependencies that can be used to initialize an Auth instance.
+ *
+ * @remarks
+ *
+ * The modular SDK enables tree shaking by allowing explicit declarations of
+ * dependencies. For example, a web app does not need to include code that
+ * enables Cordova redirect sign in. That functionality is therefore split into
+ * {@link browserPopupRedirectResolver} and
+ * {@link cordovaPopupRedirectResolver}. The dependencies object is how Auth is
+ * configured to reduce bundle sizes.
+ *
+ * There are two ways to initialize an auth instance: {@link getAuth} and
+ * {@link initializeAuth}. `getAuth` initializes everything using
+ * platform-specific configurations, while `initializeAuth` takes a Dependencies
+ * object directly, giving you more control over what is used.
+ *
  * @public
  */
 export interface Dependencies {
+  /**
+   * Which {@link Persistence} to use. If this is an array, the first
+   * Persistence that the device supports is used. The SDK will search for an
```

Comment:
Literal `Persistence` here and below?  

Lots of "will" here too :)

Reply:
Done and done